var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -54,15 +54,21 @@ class Perl(Package):
 
     resource(
         name="cpanm",
-        url="http://search.cpan.org/CPAN/authors/id/M/MI/MIYAGAWA/App-cpanminus-1.7042.tar.gz",
+        url="http://search.cpan.org/CPAN/authors/id/M/MI/MIYAGAWA"
+            "/App-cpanminus-1.7042.tar.gz",
```

Comment:
This can (and probably should) be on a single line. We exempt lines containing `http:`, `https:`, and `ftp:` from our line-length checks.